src/amo/components/SearchFilters/styles.scss

```diff
@@ -16,6 +20,11 @@
   display: block;
   line-height: 1.2;
   margin: 0 0 5px;
-  padding: 10px 10px 10px 5px;
+  padding-bottom: 10px;
+  padding-top: 10px;
   width: 100%;
+
+  [dir=rtl] & {
+    background: $grey-30 url("~ui/components/Icon/triangle-down-black.svg") no-repeat calc(10px) 50%;
```

Comment:
can't you just change `background-position` and so on to avoid duplicates?

Reply:
Oh, probably, heh. I'll do that.

Reply:
Cool, done 👍